modules/wowza/variables.tf

```diff
@@ -120,3 +120,16 @@ variable "ws_rg" {
 variable "ws_sub_id" {
   type = string
 }
+
+variable "schedules" {
+  default = []
```

Comment:
can you add a type with object example please

Reply:
Updated to 
variable "schedules" {
  type = list(object({
    name      = string
    frequency = string
    interval  = number
    run_time  = string
    start_vm  = bool
  }))
  default = []
}